src/library_html5_webgpu.js

```diff
@@ -1,11 +1,63 @@
-mergeInto(LibraryManager.library, {
+{{{ (function() {
+  // Helper functions for code generation
+  global.html5_gpu = {
+    makeImportExport: function(snake_case, CamelCase) {
```

Comment:
Is there a particular reason we need to use snake_case for these? If not it would be simpler to just use the same style for both.